src/test/Makefile.am

```diff
@@ -26,7 +26,7 @@ dist-hook:
 
 
 
-TST_LIB=libtoknz.la libdump.la libSofaStreamHandlerFile.la libSimpleTextSegmenter.la libTextSegmentConsumer.la libMeetingAnnotator.la
+TST_LIB=libtoknz.la libdump.la libSofaStreamHandlerFile.la libSimpleTextSegmenter.la libTextSegmentConsumer.la libMeetingAnnotator.la DaveDetector.la
```

Comment:
The other annotators are named "libFOO" while this new one is just "FOO", can you rename it similar to the other ones?